### PR TITLE
Updated polish localization

### PR DESCRIPTION
**Pull request by Mikihero**

### CODE REVIEW

**Comment by Mikihero** [comment]
wtf why does it show commits from months ago that I'm pretty sure were merged in a PR 💀

File: Localization/README-PL.md [hunk 1]
```diff
@@ -52,7 +51,7 @@ Jeżeli zdecydujecie się na skorzystanie z instalatora, ten - jeżeli użyty po
 To tyle, EXILED powinien być juz zainstalowany i aktywowany podczas następnego uruchomienia serwera. Pamiętajcie, że sam EXILED nie będzie robił prawie nic, aby zdobyć pluginy udajcie się na **[nasz serwer Discord](https://discord.gg/PyUkWTg)**
 - Aby zainstalować plugin wystarczy:
   - Pobrać go z [*jego* strony wydań](https://i.imgur.com/u34wgPD.jpg) (**MUSI być to plik `.dll`!**)
-  - Przenieść go do folderu: ``C:\Users\(Wasz_użytkownik)\AppData\Roaming\EXILED\Plugins`` (aby udać się tutaj wystarczy nacisnąć Win + R i wpisać `%appdata%`)
+  - Przenieść go do folderu: ``C:\Users\(Wasz_użytkownik)\AppData\Roaming\EXILED\Plugins`` (aby się tutaj dosta wystarczy wcisnąć Win + R i wpisać `%appdata%`)
```
Comment (Michal78900) [inline Localization/README-PL.md:54]: dosta -> dosta**ć**

File: Localization/README-PL.md [hunk 2]
```diff
@@ -66,11 +65,11 @@ To tyle, EXILED powinien być juz zainstalowany i aktywowany podczas następnego
...
   - Pobierzcie **`Exiled.tar.gz` kilkając [tutaj](https://github.com/Exiled-Team/EXILED/releases)** (SSH: naciśnijcie prawym przyciskiem myszy na `Exiled.tar.gz` aby skopiować link, a następnie wpiszcie: **`wget (link_do_pobrania)`**)
   - Aby wypakować pliki do obecnego folderu **``tar -xzvf EXILED.tar.gz``**
-  - Przenieście załączony plik **``Assembly-CSharp.dll``** do folderu **``SCPSL_Data/Managed``** w miejscu instalacji waszego serwera (SSH: **`mv Assembly-CSharp.dll (ścieżka_waszego_serwera)/SCPSL_Data/Managed`**).
-  - Przenieście folder **`EXILED`** do folderu **``~/.config``**. *UWAGA: Folder EXILED musi znaleźć się w folderze ``~/.config``, ***A NIE*** ``~/.config/SCP Secret Laboratory``* (SSH: **`mv EXILED ~/.config/`**)
+  - Przenieście folder **`EXILED`** do ścieżki **``~/.config``**. *UWAGA: Folder EXILED musi znaleźć się w folderze ``~/.config``, ***A NIE*** ``~/.config/SCP Secret Laboratory``* (SSH: **`mv EXILED ~/.config/`**)
+  - Przenieście folder **`SCP Secret Laboratory`** do ścieżki **``~/.config``**. *UWAGA: The folder musi znaleźć się w ``~/.config``, **A *NIE*** ``~/.config/SCP Secret Laboratory``* (SSH: **`mv SCP Secret Laboratory ~/.config/`**)
```
Comment (Michal78900) [inline Localization/README-PL.md:72]: The folder

File: Localization/README-PL.md [hunk 3]
```diff
@@ -93,21 +92,21 @@ Aby uzyskać dokładniejsze i bardziej aktualne poradniki odwiedźcie [stronę E
...
- - Jeżeli musicie wcześnie skorzystać z rzeczy, które nie zostały jeszcze zainicjalizowane przez serwer, zalecane jest aby poczekać na zdarzenie WaitingForPlayers, jeżeli z jakiegoś powodu musicie zrobić coś jeszcze wcześniej, po prostu owińcie wasz kod w pętli ``` while(!x)``` która czeka aż objekt/zmienna której potrzebujecie nie będzie miała już wartości null, a potem kontynuuje.
+ - Wasz plugin musi zawierać klasę która dziedziczy od ``Exiled.API.Features.Plugin<>``, jeżeli tak nie jest EXILED nie załaduje waszego pluginu podczas startu serwera!
+ - Gdy plugin jest załadowany kod zawarty w metodzie ``OnEnabled()`` nie czeka aż pozostałe pluginy zostaną załadowane, nie czeka aż proces uruchamiania serwera się zakończy. ***Nie czeka na nic.*** Podczas konfigurowania waszej metody ``OnEnabled()``, upewnijcie się że ***nie*** próbujecie korzystać z rzeczy, które nie zostały jeszcze zainicjalizowane przez serwer, takich jak: ``S`erverConsole.Port``, albo ``PlayerManager.localPlayer``.
```
Comment (Michal78900) [inline Localization/README-PL.md:96]: O jeden apostrof za dużo w `ServerConsole.Port`

File: Localization/README-PL.md [hunk 4]
```diff
@@ -120,15 +119,15 @@ public void JakasMetoda()
 
 public IEnumerator<float> MojaKorutyna()
 {
-    for (;;) //powtórz następujące w nieskończoność
+    for (;;) //powtórz następujący w nieskończoność
```
Comment (Michal78900) [inline Localization/README-PL.md:122]: następujący co? chyba kod

**Review by Michal78900 — CHANGES_REQUESTED** [review]
There are minor spelling mistakes

**Comment by Mikihero** [comment]
> There are minor spelling mistakes

I'll get on it later today

**Pull request merged**